_topic_map.yml

```diff
@@ -81,6 +81,8 @@ Distros: openshift-dedicated
 Topics:
 - Name: Creating your cluster
   File: creating-your-cluster
+- Name: Bring-your-own virtual private cloud
```

Comment:
Usually we try to start assemblies with a gerund (for example the "Creating" in the above assembly). Your assembly does use the "Managing" in front - can that be here as well?